packages/@aws-cdk/aws-elasticloadbalancing/lib/load-balancer.ts

```diff
@@ -400,6 +400,24 @@ export class LoadBalancer extends Resource implements IConnectable {
   }
 }
 
+/**
+ * An EC2 instance that is the target for load balancing
+ *
+ * If you register a target of this type, you are responsible for making
+ * sure the load balancer's security group can connect to the instance.
+ */
+export class InstanceTarget implements ILoadBalancerTarget {
+  /**
+   * Create a new Instance target.
+   *
+   * @param instance Instance to register to.
+   * @param connections The network connections associated with this resource.
+   */
+  constructor(public readonly instance: Instance, public readonly connections: Connections) {
```

Comment:
An `Instance` already has `Connections`, so you don't need to pass any additional ones.

Reply:
I think we might need `port` as input though! How else are we going to know what port to forward to?